packages/error-reporting/src/interfaces/manual.js

```diff
@@ -74,6 +74,10 @@ function handlerSetup(client, config) {
     }
 
     if (err instanceof ErrorMessage) {
+      var fauxError = new Error(err.message);
```

Comment:
A user does construct error message objects and, with this implementation, the user would see the stack trace at the point where they run `errors.report(errorMessage)`, not at the point where the error message object was created.

However, I think this is the correct approach.  That is, I think it would make more sense to get a trace back to the point where an error was reported.  Also, I expect the `errors.event()` method will probably be used to create messages with additional information rather than queueing messages.

If a customer has a use case where queuing is needed, we can address it at that point since we will then have a more clearly defined goal on what supporting queueing will need to look like.